src/app/components/bibliography/bibliography.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Observable, Subscription } from 'rxjs';
+// tslint:disable-next-line: max-line-length
+import { BibliographicCitation, BibliographyParserService, SortingParameters } from '../../services/xml-parsers/bibliography-parser.service';
+
+@Component({
+  selector: 'evt-bibliography',
+  templateUrl: './bibliography.component.html',
+  styleUrls: ['./bibliography.component.scss'],
+})
+export class BibliographyComponent implements OnDestroy {
+  public biblCits: BibliographicCitation[] = [];
+  private subscription: Subscription;
+  public isComplete: boolean;
+  selectAlphOrder: 'Descendent' | 'Ascendent';
+  sortingField: Observable<SortingParameters>;
+  selectSortByField: string;
+
+  constructor(
+    public bps: BibliographyParserService,
+  ) {
+    this.sortingField = this.bps.getSortingField();
+
+    this.subscription = this.bps.getBibliographicCitations().subscribe(response => {
+      this.isComplete = response.areComplete;
+      this.biblCits = response.citations;
+    });
+  }
+
+  sortBy(field: string) {
```

Comment:
It would be wonderful if you, as suggested previously, refactor this to be used as
`sortBy(field: string, mode: Asc | Desc)`